services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -33,6 +33,15 @@ class WebServerSettings(BaseCustomSettings, MixinServiceSettings, MixinSessionSe
     )
     WEBSERVER_SESSION_NAME: str = "osparc.WEBAPI_SESSION"
 
+    @cached_property
+    def base_url(self) -> str:
+        # http://webserver:8080/v0
```

Comment:
I guess this comment should be `https://webserver:8080/`

Reply:
Yep, good point. Corrected it